### PR TITLE
update indicatorfields name

### PR DESCRIPTION
**Pull request by ronykoz**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/21253

## Description
Updating indicatorField names due to server issues

### CODE REVIEW

**Comment by glicht** [comment]
@kirbles19 see release notes. they are all the same.

File: Misc/reputation-registryKey.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "contextValue": "",
     "excludedBrands": [],
     "defaultMapping": {
-        "registrykeyname": {
+        "namefield": {
```
Comment (Bars92) [inline Misc/reputation-registryKey.json]: is that on purpose?

**Comment by Bars92** [comment]
I think it should be fine. Always mind the ids and the cli name should also match. since we do not want to (temporarily) remove the system fields we must add them to content

**Comment by kirbles19** [comment]
@glicht Do we want to add section to the top of the RN: Breaking Changes. We can add this there?

**Comment by glicht** [comment]
We will need to add to the top too.

File: IndicatorFields/incidentfield-id.json [hunk 2]
```diff
@@ -11,8 +11,8 @@
   "script": "",
   "isReadOnly": false,
   "commitMessage": "",
-  "system": false,
-  "content": true,
+  "system": true,
```
Comment (Bars92) [inline IndicatorFields/incidentfield-id.json]: remove this file

File: Misc/reputation-registryKey.json (hunk 1 above)
Comment (jochman) [reply Misc/reputation-registryKey.json]: yeah, it's now pointing to `IndicatorFields/incidentfield-namefield.json`

File: IndicatorFields/incidentfield-indicatorid.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
 {
   "sortValues": null,
   "closeForm": false,
-  "cliName": "id",
+  "cliName": "indicatoridentification",
```
Comment (Bars92) [inline IndicatorFields/incidentfield-indicatorid.json]: change the file name

**Pull request merged**